src/panel.js

```diff
@@ -145,6 +144,7 @@ const IndicatorMixin = {
 
   _clearStateText: function() {
     this._thirdIndicator.text = "";
+    this._primaryIndicator.icon_name = 'audio-x-generic-symbolic';
```

Comment:
I think this line should go line 132 since it's not related to the thirdIndicator